src/main/scripts/npm-publish.bash

```diff
@@ -13,14 +13,20 @@ function err() {
     msg "$*" 1>&2
 }
 
-function main() {
+function publish() {
```

Comment:
This is not going to work. It either has to remain `main()` or you have to create a `main()` that calls `publish`.